src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java

```diff
@@ -68,13 +70,16 @@
     /**
      * This method receives an image to convert.
```

Comment:
Change This method receives an image to convert. to Receives an image to convert.